Maestro/subscriptions.json

```diff
@@ -142,54 +90,43 @@
             "/p:NotifyGitHubUsers=dotnet/coreclr-contrib",
             "/verbosity:Normal"
           ]
-        },
-        // This handler will bring the Latest CoreFX master build into WCF master
-        {
-          "maestroAction": "wcf-general",
-          "maestroDelay": "00:10:00",
-          "ScriptFileName": "build-managed.cmd",
-          "Arguments": [
-            "--",
-            "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
-            "/p:ProjectRepoOwner=dotnet",
-            "/p:ProjectRepoName=wcf",
-            "/p:ProjectRepoBranch=master",
-            "/p:NotifyGitHubUsers=dotnet/wcf-contrib",
-            "/verbosity:Normal"
-          ]
         }
-      ]
+      }
     },
+    // Update dependencies in CoreCLR release/1.0.0
```

Comment:
Some update-dependencies subscriptions include dependencies which I think would be considered internal.  For example, the CoreCLR release/1.1.0 subscription updates CoreCLR's dependency on itself.  I don't think it's worth distinguishing between subscriptions which include internal dependencies and subscriptions which don't.